models/iosxr/interfaces/iosxr_interface.yaml

```diff
@@ -0,0 +1,93 @@
+Igenerator_version: 1.0
+metadata:
+  version: 1.1
+  status:
+  - preview
+  supported_by:
+  - 'Ansible Network'
+  copyright_str: Copyright 2019 Red Hat
+  license: gpl-3.0.txt
+info:
+  network_os: iosxr
+  resource: interfaces
+  version_added: 2.9
+  short_description: 'Manage Interface on Cisco IOS network devices.'
+  description:
+  - Manage Interface on Cisco IOS-XR network devices.
+  authors:
+  - Sumit Jaiswal (@justjais)
+  extends_documentation_fragment: iosxr
+  notes:
+    - Tested against IOS-XR Version 6.1.3 on VIRL
+schema:
+  type: object
+  description: The schema used for the argspec and docstring
+  properties:
+    config:
+      type: array
+      description: The provided configuration
+      items:
+        $ref: '#/definitions/config_dict'
+    state:
+      $ref: '#/definitions/state'
+
+definitions:
+  config_dict:
+    description:
+    - The some_dict.
+    type: object
+    properties:
+      name:
+        description:
+        - Name of the interface to configure in C(type + path) format. e.g. C(GigabitEthernet0/0/0/0)
+        type: str
+        required: true
+      description:
+        description:
+        - Description of Interface being configured.
+        type: str
+      enabled:
```

Comment:
```suggestion
      enable:
```

Reply:
most other models call this enable and not enabled

Reply:
@samccann for iosxr interface earlier code, it was `enabled` so continued using the same. We need to check why it was named as `enabled` in 1st place if nothing solid I shall change it to `enable`.